codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/ApplicationTransport.java

```diff
@@ -126,7 +126,7 @@ public static ApplicationTransport createDefaultHttpApplicationTransport() {
             List<Middleware> middleware = new ArrayList();
             middleware.add((new Middleware.Builder())
                     .klass("Hearth::HTTP::Middleware::ContentLength")
-                    .step(MiddlewareStackStep.BUILD)
+                    .step(MiddlewareStackStep.SERIALIZE)
```

Comment:
Why is this moved to Serialize?

Reply:
I removed Build because it didn't make sense, it's now "validate". Content length should be after serialize.